paimon-common/src/main/java/org/apache/paimon/CoreOptions.java

```diff
@@ -310,6 +310,12 @@ public class CoreOptions implements Serializable {
                             "Max split size should be cached for one task while scanning. "
                                     + "If splits size cached in enumerator are greater than tasks size multiply by this value, scanner will pause scanning.");
 
+    public static final ConfigOption<Boolean> SCAN_SHUFFLE_BY_PARTITION =
+            key("scan.shuffle-by-partition")
```

Comment:
`streaming-read.shuffle-by-partition`?
Put this to `FlinkConnectorOptions`?

Reply:
1